### PR TITLE
Setup Html Skeleton, Css Style and Js Dom manipulation

### PR DESCRIPTION
**Pull request by chelobotix**
In this PR I fulfilled  the following:
- [x] Set up the project with webpack
- [x] Load To Do List from Object array
- [x] Add a function to iterate over the tasks array and populate an HTML list item element for each task.  
- [x] All source files (index.html, index.js, and style.css) are located in /src directory, and the distribution files were generated by webpack and served by webpack dev server from /dist folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+Clone this repository to your desired folder:
+
+```sh
+  https://github.com/chelobotix/To-Do-List.git
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
```
Comment (GzimAsani) [inline README.md:103]: - [x] Please add more information here for the user, like he needs to run npm install and npm start in order to run your project

File: README.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+### Key Features <a name="key-features"></a>
+
+- **To Do List Generate with Webpack**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🧑🏻‍💻 Live Demo <a name="live-demo"></a>
+
+<a href="https://chelobotix.github.io/Capstone-project-Protoje-concert/">Visit the Live Demo</a>
+<br/>
```
Comment (GzimAsani) [inline README.md]: - [x] Your live demo links to another project, kindly update it

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @chelobotix                        :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * I really like your readme logo :+1:  :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 



 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chelobotix** [comment]
All observations have been fixed:
Observation 1:  Please add more information here for the user, like he needs to run npm install and npm start in order to run your project
image


Observation 2: Your live demo links to another project, kindly update it
image

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @chelobotix,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**